### PR TITLE
Microservices - Add configurable items count limit

### PR DESCRIPTION
**Pull request by titkov49**

### CODE REVIEW

**Comment by titkov49** [comment]
https://user-images.githubusercontent.com/37152497/149253989-81eabab2-e816-43c0-8ab2-7f608e7df2d3.mov

**Review by Pe5h4 — APPROVED** [review]

File: src/modules/asab-config/MicroservicesContainers/MicroservicesContainer.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default (props) => {
 	const [list, setList] = useState([]);
 	const [page, setPage] = useState(1);
 	const [count, setCount] = useState(0);
+	const [limit, setLimit] = useState(10);
```
Comment (ateska) [inline src/modules/asab-config/MicroservicesContainers/MicroservicesContainer.js:14]: Default should be 20, other values 50 and 100.

**Review by ateska — CHANGES_REQUESTED** [review]
Small remark, then mergeable.

**Comment by titkov49** [comment]
Screenshot 2022-01-21 at 0 24 12
Screenshot 2022-01-21 at 0 24 24
Screenshot 2022-01-21 at 0 27 53

**Review by ateska — APPROVED** [review]

**Pull request merged**